blog/templates/blog/tag_it.html

```diff
@@ -0,0 +1,22 @@
+{% extends "blog/blogbase.html" %}
+{% block blog_content %}
+{% for object in object_list %}
+
+    
+    <div class="blog-post">
+        <h2><a class="blog-post-title"><a href="{% url "entry_view" object.id %}">{{object.title}}</a></h2>
+        {% for tag in object.tags.all %}
+           <a href="{% url 'tag_entry_view' tag.slug %}">{{ tag.name }}</a>
+        {% endfor %}
+        <p class="blog-post-meta"> {{object.posted}} 
```

Comment:
same comment about date formatting.  Everywhere you're displaying a date, make sure it's one that's consistently formatted so all dates appear the same throughout your site.